icechunk-python/src/lib.rs

```diff
@@ -890,6 +857,7 @@ impl PyIcechunkStore {
     }
 
     fn list(&self) -> PyIcechunkStoreResult<PyAsyncGenerator> {
+        #[allow(deprecated)]
```

Comment:
This is a major change to the way we do streams, punting for now and logging issue

Reply:
https://github.com/earth-mover/icechunk/issues/416